src/utils/coingecko.ts

```diff
@@ -136,7 +144,13 @@ export const fetchTokenPrices = async (
         params,
       )
         .then((data) => {
-          if (data['error'] !== undefined || data['error_code'] !== undefined) {
+          if (data === null) {
+            // Request failed (e.g., 404, network error)
+            resolve([]);
```

Comment:
Why resolve and not reject like other errors below? Or inversely, why reject below and not return []